### ISSUE TITLE
How to use loadRemoteEntry in IE 11?

### ISSUE THREAD

**Issue by leokern** [angular-architects/module-federation-plugin#30]
I'm loading microfrontends like so:

shell/main.ts:
```
import { loadRemoteEntry } from '@angular-architects/module-federation';
import { environment } from './environments/environment';

Promise.all([
  loadRemoteEntry(
    `${environment.microfrontends.mfe.entryPoint}`,
    `${environment.microfrontends.mfe.remoteName}`
  ),
])
  .catch((err) => console.error('Error loading remote entries', err))
  .then(() => import('./bootstrap'))
  .catch((err) => console.error(err));
```
When loading the app in IE 11, an error is caught (and hidden) that says:
"Object doesn't support property or method 'append'"
The error appears to occur in loadRemoteEntry at line 57 in https://github.com/angular-architects/module-federation-plugin/blob/main/packages/mf/src/utils/dynamic-federation.ts

What can I do to make this work in IE11?

Note: According to https://developer.mozilla.org/en-US/docs/Web/API/Element/append IE does not support Element.append. That does explain the error message but it doesn't help me solve this.

**Comment by leokern** [comment]
I just found https://github.com/angular-architects/module-federation-plugin/pull/24 which appears to fix the issue. Any information on if and when this may get merged?

**Comment by leokern** [comment]
Patching the generated main*.js file after the build with the following commands helps to get around this issue.
`find dist/ -name main* | xargs -I % sed "s/document\.body\.append/document\.body\.appendChild/" % > temp.js`
`find dist/ -name main* | xargs -I % mv temp.js %`

**Comment by manfredsteyer** [comment]
Thanks for pointing this out. It's merged now and will be live with version 1.2.4
